src/app/pages/storage/modules/pool-manager/components/pool-manager/tests/add-vdev-to-pool-data.ts

```diff
@@ -0,0 +1,620 @@
+    'lunid': '5000cca2700430f8',
+    'size': 12000138625024,
+    'description': '',
+    'transfermode': 'Auto',
+    'hddstandby': 'ALWAYS ON',
+    'advpowermgmt': 'DISABLED',
+    'togglesmart': true,
+    'smartoptions': '',
+    'expiretime': null,
+    'critical': null,
+    'difference': null,
+    'informational': null,
+    'model': 'HUH721212AL4200',
+    'rotationrate': 7200,
+    'type': 'HDD',
+    'zfs_guid': '15911602115397506092',
+    'bus': 'SCSI',
+    'devname': 'sdh',
+    'enclosure': {
+      'number': 0,
+      'slot': 7,
+    },
+    'supports_smart': null,
+    'pool': 'APPS',
+  },
+  {
+    'identifier': '{serial_lunid}8HG3USZH_5000cca27006f774',
+    'name': 'sdk',
+    'subsystem': 'scsi',
+    'number': 2208,
+    'serial': '8HG3USZH',
+    'lunid': '5000cca27006f774',
+    'size': 12000138625024,
+    'description': '',
+    'transfermode': 'Auto',
+    'hddstandby': 'ALWAYS ON',
+    'advpowermgmt': 'DISABLED',
+    'togglesmart': true,
+    'smartoptions': '',
+    'expiretime': null,
+    'critical': null,
+    'difference': null,
+    'informational': null,
+    'model': 'HUH721212AL4200',
+    'rotationrate': 7200,
+    'type': 'HDD',
+    'zfs_guid': '2384382266614160232',
+    'bus': 'SCSI',
+    'devname': 'sdk',
+    'enclosure': {
+      'number': 0,
+      'slot': 8,
+    },
+    'supports_smart': null,
+    'pool': 'APPS',
+  },
+  {
+    'identifier': '{serial_lunid}8HG5372H_5000cca2700947e4',
+    'name': 'sdi',
+    'subsystem': 'scsi',
+    'number': 2176,
+    'serial': '8HG5372H',
+    'lunid': '5000cca2700947e4',
+    'size': 12000138625024,
+    'description': '',
+    'transfermode': 'Auto',
+    'hddstandby': 'ALWAYS ON',
+    'advpowermgmt': 'DISABLED',
+    'togglesmart': true,
+    'smartoptions': '',
+    'expiretime': null,
+    'critical': null,
+    'difference': null,
+    'informational': null,
+    'model': 'HUH721212AL4200',
+    'rotationrate': 7200,
+    'type': 'HDD',
+    'zfs_guid': '345418775175878585',
+    'bus': 'SCSI',
+    'devname': 'sdi',
+    'enclosure': {
+      'number': 0,
+      'slot': 9,
+    },
+    'supports_smart': null,
+    'pool': 'APPS',
+  },
+  {
+    'identifier': '{serial_lunid}8HG77D9H_5000cca2700d2974',
+    'name': 'sdj',
+    'subsystem': 'scsi',
+    'number': 2192,
+    'serial': '8HG77D9H',
+    'lunid': '5000cca2700d2974',
+    'size': 12000138625024,
+    'description': '',
+    'transfermode': 'Auto',
+    'hddstandby': 'ALWAYS ON',
+    'advpowermgmt': 'DISABLED',
+    'togglesmart': true,
+    'smartoptions': '',
+    'expiretime': null,
+    'critical': null,
+    'difference': null,
+    'informational': null,
+    'model': 'HUH721212AL4200',
+    'rotationrate': 7200,
+    'type': 'HDD',
+    'zfs_guid': '12869578439632579419',
+    'bus': 'SCSI',
+    'devname': 'sdj',
+    'enclosure': {
+      'number': 0,
+      'slot': 10,
+    },
+    'supports_smart': null,
+    'pool': 'APPS',
+  },
+] as unknown as Disk[];
```

Comment:
Would be good to clean up this file and remove things that are not really needed.